### PR TITLE
When user clicks on one of the date button filter button is not highlighted

### PR DESCRIPTION
**Pull request by jbargu**
Jira ticket: [SCI-3226](https://biosistemika.atlassian.net/browse/SCI-3226)

### What was done
Added background color to buttons, switched from/to date fields, enabled
clear filter functionality for date field.

### CODE REVIEW

**Comment by okriuchykhin** [comment]
Activities are always fetched in reverse order, from new to old, so 'from_date' > 'to_date'

File: app/assets/javascripts/global_activities/side_pane.js [hunk 1]
```diff
@@ -191,20 +198,22 @@ $(function() {
   $('#calendar-to-date').on('dp.change', function(e) {
     var dateContainer = $('.ga-side .date-selector.filter-block');
     if (!updateRunning) {
-      $('#calendar-from-date').data('DateTimePicker').minDate(e.date);
+      $('#calendar-from-date').data('DateTimePicker').maxDate(e.date);
```
Comment (okriuchykhin) [inline app/assets/javascripts/global_activities/side_pane.js]: Activities are fetched in reverse chronological order

File: app/views/global_activities/_side_filters.html.erb [hunk 2]
```diff
@@ -30,12 +30,12 @@
     <h4 class="title">
       <%= t('global_activities.index.custom_dates_label') %>
     </h4>
-    <div class="to">
-      <%= render partial: "date_picker", locals: { id: 'to-date', use_current: 'false', placeholder: t('global_activities.index.to_label'), label: nil } %>
+    <div class="from">
```
Comment (okriuchykhin) [inline app/views/global_activities/_side_filters.html.erb]: Activities are fetched in reverse chronological order

File: app/assets/javascripts/global_activities/side_pane.js [hunk 3]
```diff
@@ -191,20 +198,22 @@ $(function() {
...
       reloadActivities();
+      resetHotButtonsBackgroundColor();
     }
   });
 
   $('#calendar-from-date').on('dp.change', function(e) {
     var dateContainer = $('.ga-side .date-selector.filter-block');
     if (!updateRunning) {
-      $('#calendar-to-date').data('DateTimePicker').maxDate(e.date);
+      $('#calendar-to-date').data('DateTimePicker').minDate(e.date);
```
Comment (okriuchykhin) [inline app/assets/javascripts/global_activities/side_pane.js]: Activities are fetched in reverse chronological order

File: app/assets/javascripts/global_activities/side_pane.js (hunk 1 above)
Comment (jbargu) [reply app/assets/javascripts/global_activities/side_pane.js]: Ok, I'm tad confused here about the names. Not really sure if this is something intuitive for the user. And in epics it also is FROM-TO. Should then also the filter label be reversed as it was (`Activity created: 31.01.2019-01.01.2019`)?
I'm willing to bet someone's gonna burn himself on the names down the road. :D

Comment (okriuchykhin) [reply app/assets/javascripts/global_activities/side_pane.js]: User is not supposed to look into HTML attributes. User will still see 'start' and 'end' date.

Comment (jbargu) [reply app/assets/javascripts/global_activities/side_pane.js]: I meant for developer fixing bugs. For user it's intuitive to think 2 date fields will be from-to, e.g. you'll probably never say 'until today from yesterday', but what do I know, I'm just a developer.

Comment (okriuchykhin) [reply app/assets/javascripts/global_activities/side_pane.js]: where did you take 'until today from yesterday' from? What is illogical in 'from today to previous Sunday' as we use reverse chronological order?

**Comment by jbargu** [comment]
Please see the revised version.

**Pull request merged**